src/Kreta/TaskManager/Domain/Model/Project/Task/TaskTitleCannotBeEmptyException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Kreta package.
+ *
+ * (c) Beñat Espiña <benatespina@gmail.com>
+ * (c) Gorka Laucirica <gorka.lauzirika@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Kreta\TaskManager\Domain\Model\Project\Task;
+
+use Kreta\SharedKernel\Domain\Model\Exception;
+
+class TaskTitleCannotBeEmptyException extends Exception
+{
+    public function __construct()
+    {
+        $this->message = 'Task title cannot be empty';
```

Comment:
Is more defensive if we call the parent constructor, this approach can generate side effects